From e09117a7ac1a978b7d7f9f1815af256da939a7c2 Mon Sep 17 00:00:00 2001
From: liuzhensong <liuzhensong@loongson.cn>
Date: Thu, 12 Aug 2021 18:34:05 +0800
Subject: [PATCH 21/30] binutils: Add Base on desc.

  bfd/elfxx-loongarch.c
  gdb/nat/loongarch-linux-watch.c
  gdb/nat/loongarch-linux-watch.h
  include/elf/loongarch.h
---
 bfd/elfxx-loongarch.c   | 2 ++
 include/elf/loongarch.h | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/bfd/elfxx-loongarch.c b/bfd/elfxx-loongarch.c
index eb3b4523a65..3660049fe00 100644
--- a/bfd/elfxx-loongarch.c
+++ b/bfd/elfxx-loongarch.c
@@ -2,6 +2,8 @@
    Copyright (C) 2021 Free Software Foundation, Inc.
    Contributed by Loongson Ltd.
 
+   Based on RISC-V target.
+
    This file is part of BFD, the Binary File Descriptor library.
 
    This program is free software; you can redistribute it and/or modify
diff --git a/include/elf/loongarch.h b/include/elf/loongarch.h
index 3289bc87545..1c873fc1f34 100644
--- a/include/elf/loongarch.h
+++ b/include/elf/loongarch.h
@@ -3,6 +3,8 @@
 
    This file is part of GNU Binutils.
 
+   Based on RISC_V target.
+
    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 3 of the license, or
-- 
2.30.1

